### PR TITLE
Update CyberARK EPM docs

### PR DESCRIPTION
**Pull request by paulina-kruczek-sumo**
## Purpose of this pull request

This pull request (PR) updates CyberARK EPM docs with the new endpoint information.

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cyberark-source.md [hunk 1]
```diff
@@ -86,6 +87,7 @@ To configure a CyberArk EPM Source, follow the steps below:
 1. **Rate Limit C2C**. This option removes the request limitations imposed on the CyberArk C2C source, allowing it to make as many requests as possible. By default, the request limitations are set to 5 requests per minute for admin audit events and 1000 requests per 5 minutes for detailed raw and aggregated policy audit events. If you need to make more requests than the default limits allow, you can contact the CyberArk support team to request the limitations be removed.
 1. **Collect detailed raw events**. This option enables the CyberArk C2C Source to collect detailed raw events from the CyberArk EPM (Endpoint Privilege Manager).
 1. **Collect aggregated policy audit events**. This option enables the C2C Source to collect aggregated policy audit events from the CyberArk EPM (Endpoint Privilege Manager).
+1. **Collect policy audit raw events**. This option enables the C2C Source to collect policy audit raw events from the CyberArk EPM (Endpoint Privilege Manager).
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cyberark-source.md:90]: No need to define EPM acronym again (it's defined in line 12).

```suggestion
1. **Collect policy audit raw events**. This option enables the C2C Source to collect policy audit raw events from the CyberArk EPM.
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
Minor fix

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cyberark-source.md [hunk 2]
```diff
@@ -11,15 +11,16 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
 
-The integration with CyberArk EPM's API allows for retrieving administrative, detailed raw, and policy audit events from every set in the environment. The [API documentation](https://docs.cyberark.com/Product-Doc/OnlineHelp/EPM/Latest/en/Content/LandingPages/LPDeveloper.htm) provides guidance on accessing and utilizing this information. This integration facilitates retrieving various audit events, including administrative actions, policy violations, and application usage, to generate alerts, reports, and remediation actions that enhance the organization's security posture.
+The integration with CyberArk EPM's API allows for retrieving administrative, detailed raw, policy audit and policy audit raw events from every set in the environment. The [API documentation](https://docs.cyberark.com/Product-Doc/OnlineHelp/EPM/Latest/en/Content/LandingPages/LPDeveloper.htm) provides guidance on accessing and utilizing this information. This integration facilitates retrieving various audit events, including administrative actions, policy violations, and application usage, to generate alerts, reports, and remediation actions that enhance the organization's security posture.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cyberark-source.md:14]:
```suggestion
The integration with CyberArk EPM's API allows for retrieving administrative, detailed raw, policy audit, and policy audit raw events from every set in the environment. The [API documentation](https://docs.cyberark.com/Product-Doc/OnlineHelp/EPM/Latest/en/Content/LandingPages/LPDeveloper.htm) provides guidance on accessing and utilizing this information. This integration facilitates retrieving various audit events, including administrative actions, policy violations, and application usage, to generate alerts, reports, and remediation actions that enhance the organization's security posture.
```

**Comment by paulina-kruczek-sumo** [comment]
@kimsauce We waited with the documentation until changes release. Now it can be merged.

**Comment by kimsauce** [comment]
> @kimsauce We waited with the documentation until changes release. Now it can be merged.

perfect! thank you for the update.

**Pull request merged**